packages/teams-js/src/internal/communication.ts

```diff
@@ -257,6 +264,90 @@ export function sendMessageToParentAsync<T>(
   });
 }
 
+type SimpleType = string | number | boolean | null | undefined | SimpleType[];
+
+function isSerializable(arg: unknown): arg is ISerializable {
```

Comment:
>isSerializable [](http://example.com/codeflow?start=9&length=14)

I think this would more appropriately live next to the definition of `ISerializable` since its conceptually part of the type definition.